### ISSUE TITLE
Amnesia -- itemized deduction surtax checkboxes

### ISSUE THREAD

**Issue by MattHJensen** [OpenSourcePolicyCenter/webapp-public#277]
@feenberg identified a bug having to do with surtax on itemized deduction benefits. It can be reproduced like this:

1. Create a reform with these provisions:
    - surtax on itemized deduction benefits is 1.0
    - credit rate is 0.0. 
    - All of the deductions are unselected. 
2. Run calculations (e.g., http://www.ospc.org/taxbrain/4558/) .
    - Note that the revenue impact is 0, as it should be, since we unselected all of the deductions. 
3. Click edit parameters (.e.g., http://www.ospc.org/taxbrain/edit/4558/?start_year=2013)
4. Select all of the itemized deduction check boxes:
![image](https://cloud.githubusercontent.com/assets/8301092/16726660/5d7561d0-472b-11e6-9005-3f5c2769d1ce.png)
5. Run calculations (e.g., http://www.ospc.org/taxbrain/4559/)
    - Note that the revenue impact is still 0, but it should be large and positive because we eliminated all itemized deductions by taxing their benefits at 100%.
6. Click edit parameters (e.g., http://www.ospc.org/taxbrain/edit/4559/?start_year=2013)
     - Note that the deductions are still unselected, even though we selected them in step 4. 

cc @talumbau @martinholmer

**Comment by talumbau** [comment]
I can reproduce this. The `user_mods` for 4558 is below, but then when you edit a scenario like this and switch the flags to 'True', the web application does not accept them.

4558:
```
user_mods is  {'_ID_BenefitSurtax_Switch': [[False, False, False, False, False, False, False]], '_ID_BenefitSurtax_trt': [1.0], '_ID_BenefitSurtax_crt': [0.0]} 
```

**Comment by talumbau** [comment]
An update here. When executing step 4  ("check all boxes to 'on' and then hit 'submit'"), the POST comes back with the value of `False` for the check boxes, even though the user clicked them to `True`.

If you edit a reform policy like taxbrain/4558, and the directly hit 'Submit' again, there is no info on the 6 boolean switches in the POST, and we get the value of the flags (in this case, all `False`) from the form, which all works as expected.

It's unclear why the POST values come back as `False`.

**Comment by talumbau** [comment]
closed via #281
